### ISSUE TITLE
[BUG] Escaped HTML in post_meta data date

### ISSUE THREAD

**Issue by rathod-sahaab** [adityatelange/hugo-PaperMod#1344]
**Describe the bug**
On local builds (latest hugo everything works fine, remote escape date html

 - Device/Os: [firefox]
 - Type: [Desktop/Mobile]
 - Browser and version [e.g. firefox]:
 - Hugo Version [ latest expected]:
 - Theme Version [master]:

**Steps to reproduce the behavior:**
1. Go to [my blog](https://abhay.rs/blog/posts/backend-intermediate-0/)
2. See post meta data date section

**Expected behavior**:
HTML as HTML not escaped

**Repo/Source where this issue can be reproduced**:
[rathod-sahaab/blog](https://github.com/rathod-sahaab/blog/tree/dev/content/posts/backend-intermediate-0)

**Screenshots**
Remote:
![image](https://github.com/adityatelange/hugo-PaperMod/assets/31929374/93a0fef0-36a8-4e28-93b1-b38370152eea)

Local:
![image](https://github.com/adityatelange/hugo-PaperMod/assets/31929374/19284d8d-bb21-4057-89e6-49da56b92e35)

**Additional context**
- using action [peaceiris/actions-hugo](https://github.com/peaceiris/actions-hugo
)

**Comment by adityatelange** [comment]
- #1325

**Comment by Aryaman73** [comment]
Hello! I've been reencountering this issue on my paper mod website (link: https://www.aryamans.me). I'm also using the peaceiris github action to deploy my website using GH pages.

I'm a bit lost on what the best way to resolve the issue is - I've tried updating the hugo versions of my GitHub page to no avail. Am I misconfiguring my page, or has this issue reappeared somehow?

Repo Link: https://github.com/Aryaman73/aryaman73.github.io/tree/master
GH Page Link: https://github.com/Aryaman73/aryaman73.github.io/blob/master/.github/workflows/gh-pages.yml

**Comment by guiyumin** [comment]
same here

https://notes.ossking.com/


Can anyone help us? Thanks

![image](https://github.com/adityatelange/hugo-PaperMod/assets/43506282/0c1caa81-1093-4fc5-a87e-e36443358c40)

**Comment by danaoairuike** [comment]
> Hello! I've been reencountering this issue on my paper mod website (link: https://www.aryamans.me). I'm also using the peaceiris github action to deploy my website using GH pages.
> 
> I'm a bit lost on what the best way to resolve the issue is - I've tried updating the hugo versions of my GitHub Action to no avail. Am I misconfiguring my page, or has this issue reappeared somehow?
> 
> Repo Link: https://github.com/Aryaman73/aryaman73.github.io/tree/master GH Page Link: https://github.com/Aryaman73/aryaman73.github.io/blob/master/.github/workflows/gh-pages.yml

Hey man, just use the master branch of PaperMod, and it has fixed the bug.

**Comment by danaoairuike** [comment]
> Hey man, just use the master branch of PaperMod, and it has fixed the bug.

Hey man, just use the master branch of PaperMod, and it has fixed the bug.

**Comment by danaoairuike** [comment]
> same here
> 
> https://notes.ossking.com/
> 
> Can anyone help us? Thanks
> 
> ![image](https://private-user-images.githubusercontent.com/43506282/304607366-0c1caa81-1093-4fc5-a87e-e36443358c40.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5NDg0NDksIm5iZiI6MTcwNzk0ODE0OSwicGF0aCI6Ii80MzUwNjI4Mi8zMDQ2MDczNjYtMGMxY2FhODEtMTA5My00ZmM1LWE4N2UtZTM2NDQzMzU4YzQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDIyMDIyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiMDc3ZjRhNjY0YzFlMTAyZTZjMzAxMTdkMzA4ZmZkM2RmNTUyMmM5ZjZhMGQxYzM3NDhhNWQ4OTI3MzVmODImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1sJinwxGRfwWd4fymuHaNexVR7yHyEd2MQ6M783YgIs)

Hey man, just use the master branch of PaperMod, and it has fixed the bug.

**Comment by Aryaman73** [comment]
@guiyumin (and to anyone encountering this issue), @danaoairuike is correct - update the PaperMod theme. The best way to do this is to just `rm -rf ./themes` and then clone it again using [these instructions](https://github.com/adityatelange/hugo-PaperMod/wiki/Installation) - the command I used was:

```bash
git clone https://github.com/adityatelange/hugo-PaperMod themes/PaperMod --depth=1
```

---

I'm still encountering some other issues dealing with [this error](https://github.com/gohugoio/hugo/issues/7087) - it *may* be related to this issue. If you do encounter this issue, I got around it by getting rid of the build step in the github workflow file, and running `hugo build --minify` locally and pushing those files. It's obviously not ideal, but at least my site is functioning again!
